packages/frontend/src/components/dialogs/AlertDialog.tsx

```diff
@@ -39,7 +39,7 @@ export default function AlertDialog({
       </DialogBody>
       <DialogFooter>
         <FooterActions>
-          <FooterActionButton onClick={onClick}>
+          <FooterActionButton styling='primary' onClick={onClick}>
```

Comment:
I'm not sure about making those "dismiss" buttons the primary color. It makes it seem like some action is about to be performed.
There are a few more such places.

Reply:
Same topic as above. Are single buttons "primary" actions or not?